rclcpp_components/src/component_manager.cpp

```diff
@@ -178,6 +178,17 @@ ComponentManager::OnLoadNode(
                     "Extra component argument 'use_intra_process_comms' must be a boolean");
           }
           options.use_intra_process_comms(extra_argument.get_value<bool>());
+        } else if (extra_argument.get_name() == "domain") {
```

Comment:
I think this makes the argument a bit more descriptive:
```suggestion
        } else if (extra_argument.get_name() == "domain_id") {
```

Reply:
I fixed this when I moved the code to the linked PR.